CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # pylhc Changelog
 
+## Version 0.7.4
+
+Changes in Machine Settings Info
+ - Default behaviour for no knobs given: extract None.
+ - Old behaviour of extracting all restored by giving `knobs = ["all"]`
+ - Option `["default"]` available for default knobs.
```

Comment:
should be without the brackets, shouldn't it? Or does it refer to what should be passed to the API (not commandline)?